boxes.controller.js

```diff
@@ -0,0 +1,45 @@
+const { Box } = require('./model');
+
+exports.getAllBoxes = async (ctx) => {
+  try {
+    ctx.body = await Box.find({});
+  } catch (err) {
+    console.log(err); // eslint-disable-line
+    ctx.status = 500;
+  }
+};
+
+exports.getBox = async (ctx) => {
+  try {
+    [ctx.body] = await Box.find({ _id: ctx.params.id });
+  } catch (err) {
+    console.log(err); // eslint-disable-line
+    ctx.status = 500;
+  }
+};
+
+exports.changeStatus = async (ctx) => {
+  const {
+    dataLeft = 0, minsLeft = 0, expiration = '', comments = '', simType,
+  } = ctx.request.body;
+  try {
+    const box = await Box.findById(ctx.params.id);
+    await Box.findByIdAndUpdate(ctx.params.id, {
+      // It's getting a deprecation warning, but if I change
+      // it to findOneAndUpdate I get the warning anyway 🤷‍
+      updated: new Date().toISOString(),
+      full: !box.full,
```

Comment:
You can avoid line 26 `findById` by using bitwise operators

```
$bit: {
  full: { xor: 1 }
}
```

It's a bit too much maybe, but allows you to avoid a pre-query.